src/components/movieTitle/movieTitle.tsx

```diff
@@ -0,0 +1,27 @@
+import './movieTitle.css';
+
+interface MovieTitleProps {
+  imageUrl: string;
+  movieName: string;
+  releaseYear: number;
+  relevantGenres: Genre[];
+  onMovieSelect: () => void;
+}
+
+const MovieTitle = (props: MovieTitleProps) => {
```

Comment:
The name of component is MovieTile (not Title)
The same with props (described for sortControl)